BadgeBLE.md

```diff
@@ -50,3 +58,130 @@ The "mode" bytes are a combination of two 4 bit values. The high nibble describe
 | 0x07 | "picture"         |
 | 0x08 | "laser"           |
 
+### Next-Gen profile
+
+ 16-bit Service/characteristic:
+
+- Service-UUID: 0xF055 (128-bit equivalent: 0000f055-0000-1000-8000-00805f9b34fb)
+- Characteristic: 0xF056 (128-bit equivalent: 0000f056-0000-1000-8000-00805f9b34fb)
+  - Write property: to receive controls and configs
+  - Notify property: to return error codes
+  - Read property: to be implemented
+
+#### Protocol
+
+The next-gen characteristic takes message varies in length (0 - 512 bytes). The first byte is function/command code and the rest of the message is parameters for the corresponding function/command.
```

Comment:
Do you mean this?

The next-gen characteristic accepts messages of varying lengths (0 to 512 bytes). The first byte represents the function/command code, while the remaining bytes contain parameters for the corresponding function/command.